Quadriga/src/main/java/edu/asu/spring/quadriga/dto/UploadTransfomationFilesDTO.java

```diff
@@ -0,0 +1,56 @@
+package edu.asu.spring.quadriga.dto;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.Column;
+@Entity
+@Table(name = "trns_files")
+public class UploadTransfomationFilesDTO {
```

Comment:
You class needs an id field. And you probably also want to store the file names of transformation and mapping file. Otherwise, how would you know what files to load?